packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatBackedChatAdapter.ts

```diff
@@ -83,8 +89,27 @@ export class CallWithChatBackedChatAdapter implements ChatAdapter {
         return this.callWithChatAdapter.off(event, listener);
     }
   };
-  public updateMessage = async (messageId: string, content: string, metadata?: Record<string, string>): Promise<void> =>
-    await this.callWithChatAdapter.updateMessage(messageId, content, metadata);
+
+  // cannot use arrow function because CC won't remove paramters properly
+  // have to bind this since the scope of 'this' is lost when the function is passed as a callback
+  updateMessageHandler = async function (
+    this: CallWithChatBackedChatAdapter,
```

Comment:
would using changing parameter name `this` to some else work with arrow function (eg `thisRef`? And in the function call `thisRef.callWithChatAdapter.updateMessage(...`

Reply:
<img width="848" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/19164206-df5a-4e5b-a10e-89f1f2a33015">

actually this doesn't seem to work, `thisRef` is treated as normal parameter and we can no longer use `.bind(this)` to bind the local context